src/libraries/System.Text.Json/tests/Serialization/TestClasses.SimpleTestClassWithFields.cs

```diff
@@ -0,0 +1,511 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Linq;
+using Xunit;
+
+namespace System.Text.Json.Serialization.Tests
+{
+    public class SimpleTestClassWithFields : ITestClass
```

Comment:
I believe you'll need to add these test classes to `ReadSuccessCases` and `WriteSuccessCases` in TestData.cs (unless I missed where they are used).